connectors/services/job_execution.py

```diff
@@ -27,31 +49,77 @@ class JobExecutionService(BaseService):
     def __init__(self, config):
         super().__init__(config)
         self.idling = self.service_config["idling"]
-        self.concurrent_syncs = self.service_config.get(
-            "max_concurrent_syncs", DEFAULT_MAX_CONCURRENT_SYNCS
+        self.max_concurrent_content_syncs = load_max_concurrent_content_syncs(
+            self.service_config
+        )
+        self.max_concurrent_access_control_syncs = (
```

Comment:
Just curious, why the outer parenthesis? added automatically by `make autoformat`?